lte/gateway/c/core/oai/tasks/amf/nas5g_message.cpp

```diff
@@ -320,9 +325,11 @@ int nas5g_message_encode(
         _nas5g_message_plain_encode(buffer, &msg->header, &msg->plain, length);
 
     OAILOG_INFO(
-        LOG_AMF_APP, "Encoded msg(nas5g) id: [%x]-name [%s]",
+        LOG_AMF_APP, "Encoded msg(nas5g) id: [%lu]-name [%s]",
```

Comment:
same comment here regarding `%u` formatting

Reply:
the actual intention was to print hex value for message_type.I have reverted back to hex print.